metafix/src/test/java/org/metafacture/metafix/MetafixMethodTest.java

```diff
@@ -305,10 +305,32 @@ public void shouldNotTrimRepeatedField() {
         );
     }
 
+    @Test
+    public void shouldNotTrimIndexedArray() {
```

Comment:
It's not an "indexed array", is it? The entity is missing the array marker.

Reply:
Right, thanks, tweaked and added test in 9b8fa39.